internal/zinc-scripted/src/test/scala/sbt/internal/inc/IncHandler.scala

```diff
@@ -482,6 +485,12 @@ case class ProjectStructure(
       ()
     }
 
+  def checkExists(i: IncState, className: String, name: String): Future[Unit] =
```

Comment:
This name is a little vague. It would be more clear if it was something like checkNameExistsInClass. I found the test cases at the bottom confusing to read without looking at the source code.

Reply:
I agree now that you've mentioned it, but in the interest of reducing the barrier-to-merge I'm going to merge this as-is, and split your comment to a new issue.